### PR TITLE
Adjust media plugins to new interfaces

### PR DESCRIPTION
**Pull request by mzanetti**
nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.
@Boernsman can you do some basic testing on the ones you have available please

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

- [x] If you added a new plugin, should it be added to nymea-plugins-all or nymea-plugins-maker?

### CODE REVIEW

**Comment by mzanetti** [comment]
Acking @Boernsman 's changes. Please review mine.

File: denon/integrationplugindenon.json [hunk 1]
```diff
@@ -255,28 +255,12 @@
...
                             "displayName": "Increase volume",
-                            "name": "increaseVolume",
-                            "paramTypes": [
-                                {
-                                    "id": "765c7e2a-9eb6-46fc-a880-4e96c81f8d1e",
-                                    "displayName": "Step",
-                                    "name": "step",
-                                    "type": "int"
-                                }
-                            ]
```
Comment (Boernsman) [inline denon/integrationplugindenon.json:266]: Why do you remove the step size on this thingClass, but added it on other thingClasses (e.g. Heos)

Comment (Boernsman) [reply denon/integrationplugindenon.json:266]: I'ld rather add step size to every in-/decrease volume action, I guess this feature is mainly used by building automation integrators and having the possibility to configure the step size on a button press will be handy.

**Review by Boernsman — CHANGES_REQUESTED** [review]

File: denon/integrationplugindenon.json (hunk 1 above)
Comment (mzanetti) [reply denon/integrationplugindenon.json:266]: thanks for the help. Adjusted min/max checking on top of your commit.

**Comment by mzanetti** [comment]
merged

**Pull request closed without merging**